mwclient/client.py

```diff
@@ -440,13 +530,38 @@ def raw_api(self, action, http_method='POST', *args, **kwargs):
             raise errors.InvalidResponse(res)
 
     def raw_index(self, action, http_method='POST', *args, **kwargs):
-        """Sends a call to index.php rather than the API."""
+        """Sends a call to index.php rather than the API.
+
+        Args:
+            action (str): The MediaWiki API action to perform.
+            http_method (str): The HTTP method to use in the request.
+
+        Returns:
+            The API response.
+        """
```

Comment:
as with `raw_api`, this can raise anything `raw_call` can. though interestingly there's no way to set `retry_on_error` false here - you're stuck with it set true.

Reply:
Added the exceptions from `raw_call` here.